cmd/lssmbios/main.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 DigitalOcean.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command lssmbios accesses and displays SMBIOS data.
+package main
+
+import (
+	"log"
+
+	"github.com/digitalocean/go-smbios/smbios"
+)
+
+func main() {
+	// Find SMBIOS data in operating system-specific location.
+	rc, err := smbios.Stream()
+	if err != nil {
+		log.Fatalf("failed to open stream: %v", err)
+	}
+	// Be sure to close the stream!
+	defer rc.Close()
+
+	// Decode SMBIOS structures from the stream.
+	d := smbios.NewDecoder(rc)
+	ss, err := d.Decode()
+	if err != nil {
+		log.Fatalf("failed to decode structures: %v", err)
+	}
+
+	for _, s := range ss {
+		log.Println(s)
```

Comment:
`fmt.Println` since that's the output of your program and should thus go to `stdout`

Reply:
Done.